lib/qa/authorities/linked_data/config/term_config.rb

```diff
@@ -70,13 +70,31 @@ def term_results_id_ldpath
         Config.config_value(term_results, :id_ldpath)
       end
 
+      # Return results id_predicates
+      # @return [Array<String>] the configured predicate to use to extract the id from the results
+      def term_results_id_predicates
+        return @pred_ids unless @pred_ids.blank?
+        pred = Config.predicate_uri(term_results, :id_predicate)
+        @pred_ids = [pred] unless pred.blank?
+        return @pred_ids unless @pred_ids.blank?
+        @pred_ids = id_predicates_from_ldpath
+      end
+
       # Return results id_predicate
       # @return [String] the configured predicate to use to extract the id from the results
-      def term_results_id_predicate
-        return @pred_id unless @pred_id.blank?
-        @pred_id = Config.predicate_uri(term_results, :id_predicate)
-        return @pred_id unless @pred_id.blank?
-        @pred_id = pred_id_from_ldpath
+      # NOTE: Customizations using this method should be updated to use `term_results_id_predicates` which returns [Array<String>] of
+      #       id predicates.  This method remains for backward compatibility only but may cause issues if used in places expecting an Array
+      def term_results_id_predicate(suppress_deprecation_warning: false)
```

Comment:
This remains for backward compatibility only.  It is not used by the QA code and will be removed at the next major release.